utils/soca/fig_gallery/marine_vrfy_display/index_vrfy_marine.html

```diff
@@ -0,0 +1,853 @@
+						<div class="dropdown-menu">
+        					        <button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'insitu', 'profile', 'argo', 'Salt')">ARGO Salinity</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'insitu', 'profile', 'argo', 'Temp')">ARGO Temperature</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'insitu', 'profile', 'tesac_salinity', 'Salt')">TESAC Salinity</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'insitu', 'profile', 'tesac', 'Temp')">TESAC Temperature</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'insitu', 'profile', 'trkob_salinity', 'Salt')">TRKOB Salinity</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'insitu', 'profile', 'trkob', 'Temp')">TRKOB Temperature</button>
+						</div>
+					</div>
+					<div class="dropdown-menu">
+        					<button class="dropbtn">SeaIce Obs</button>
+        					<div class="dropdown-menu">
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'icec', 'amsr2', 'north', '')">AMSR-2 (Arctic)</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'icec', 'amsr2', 'south', '')">AMSR-2 (Antarctic)</button>
+						</div>
+					</div>
+      					<div class="dropdown-menu">
+        					<button class="dropbtn">SST Obs</button>
+        					<div class="dropdown-menu">
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'sst', 'abi', 'g16', '')">ABI (GOES-16)</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'sst', 'abi', 'g17', '')">ABI (GOES-17)</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'sst', 'ahi', 'h08', '')">AHI (HIMAWARI-8)</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'sst', 'avhrr', 'ma', '')">AVHRR (METOP-A)</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'sst', 'avhrr', 'mb', '')">AVHRR (METOP-B)</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'sst', 'viirs', 'n20', '')">VIIRS (NOAA-20)</button>
+							<button class="dropbtn" id="hofx" onclick="showImagesDiags('observation_scatter_plots', 'sst', 'viirs', 'npp', '')">VIIRS (S-NPP)</button>
+						</div>
+					</div>
+			        </div>
+			    </div>
+                            
+                        </ul>
+		</div>
+	</div>
+	<div id="main">
+		<h1><font color="blue">Marine Figures</font></h1>
+		<h2>Add path to your /vrfy directory:</h2>

```

Comment:
update this, maybe give an example?

Reply:
I added some clarifying text about what should be included in the path, depending on whether the vrfy figures are in the user's COMROOT or from HPSS.